server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/GetMessagesMethodStepdefs.java

```diff
@@ -446,6 +450,12 @@ public void assertAttachmentsHasSize(int numberOfAttachments) throws Throwable {
         assertThat(jsonPath.<List<Object>>read(ATTACHMENTS)).hasSize(numberOfAttachments);
     }
 
+    @Then("^the list of attachments of the message contains (\\d+) attachments with cid \"([^\"]*)\"?$")
```

Comment:
You don't need to have several attachments as you are matching only one CID.
So should be "contains 1 attachment"